Equilobe.DailyReport.SL/ConfigurationService.cs

```diff
@@ -57,5 +54,15 @@ public string GetBitBucketApiClientUrl()
         {
             return ConfigurationManager.AppSettings["bitBucketApiClientUrl"];
         }
+
+        public string GetDashboardDataSyncScriptPath()
+        {
+            return ConfigurationManager.AppSettings["dashboardDataSyncScriptPath"];
+        }
+
+        public string GetPowershell()
```

Comment:
What is this for? Don't we have the script from GetDashboardDataSyncScriptPath?

Reply:
I found that I cannot run the script on its own. I have to start a scheduled task with the program Powershell.exe having the arguments the script and its arguments;